corc-cascading/src/test/java/com/hotels/corc/cascading/FieldsTypeUtilsTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (C) 2015-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.corc.cascading;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import cascading.tuple.Fields;
+import org.apache.hadoop.hive.common.type.HiveChar;
+import org.apache.hadoop.hive.common.type.HiveVarchar;
+import org.apache.hadoop.hive.ql.io.sarg.PredicateLeaf;
+import org.apache.hadoop.hive.serde2.io.HiveDecimalWritable;
+import org.junit.Test;
+
+import java.sql.Date;
+import java.sql.Timestamp;
+
+import static com.hotels.corc.cascading.FieldsTypeUtils.toType;
+
+public class FieldsTypeUtilsTest {
+
+    @Test
+    public void intToLongType(){
+        Fields ONE = new Fields("A", Integer.class);
```

Comment:
Java convention is to use all upper case only for constants (typically static, final and immutable). The `Fields` object here isn't constant and the name `ONE` is a bit misleading and is a hangover from where it originally came from. I'd suggest renaming this to just `fields` (applies to all instances below too).